ldaptor/protocols/ldap/distinguishedname.py

```diff
@@ -11,15 +11,22 @@
 escapedChars_trailing = r' #'
 
 
+def _to_bytes(s):
```

Comment:
this looks like duplicate code...and is generic/utility code...so I think that it can be moved outside of this module.

not sure why six does not have yet this helper :)